File: Python/1018.py
i = int(input())

print(i)
print("{} nota(s) de R$ 100,00".format(int(i/100)))
rest = (i%100)
print("{} nota(s) de R$ 50,00".format(int(rest/50)))
rest = (rest%50)
print("{} nota(s) de R$ 20,00".format(int(rest/20)))
rest = (rest%20)
print("{} nota(s) de R$ 10,00".format(int(rest/10)))
rest = (rest%10)
print("{} nota(s) de R$ 5,00".format(int(rest/5)))
rest = (rest%5)
print("{} nota(s) de R$ 2,00".format(int(rest/2)))
rest = (rest%2)
print("{} nota(s) de R$ 1,00".format(int(rest/1)))

File: JavaScript/1018.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let N = parseInt(lines[0]);

if (0 < N < 1000000){
    console.log(N);
    console.log(`${parseInt(N/100)} nota(s) de R$ 100,00`, );
    N%=100;
    console.log(`${parseInt(N/50)} nota(s) de R$ 50,00`);
    N%=50;
    console.log(`${parseInt(N/20)} nota(s) de R$ 20,00`);
    N%=20;
    console.log(`${parseInt(N/10)} nota(s) de R$ 10,00`);
    N%=10;
    console.log(`${parseInt(N/5)} nota(s) de R$ 5,00`);
    N%=5;
    console.log(`${parseInt(N/2)} nota(s) de R$ 2,00`);
    N%=2;
    console.log(`${parseInt(N/1)} nota(s) de R$ 1,00`);
    N%=1;
};